src/__test__/components/data-exploration/generic-gene-table/SelectionActions.test.jsx

```diff
@@ -151,12 +145,12 @@ describe('SelectionIndicator', () => {
       </Provider>,
     );
 
-    const button = component.find(Button);
+    const buttons = component.find(Button);
 
     // There should be one button.
-    expect(button.length).toEqual(1);
+    expect(buttons.length).toEqual(2);
 
-    button.simulate('click');
+    buttons.at(0).simulate('click');
     expect(mockCallback).toHaveBeenCalledTimes(1);
```

Comment:
Can you add a test in `SelectionActions.test.jsx` to check if the modal for `pathway analysis` is correctly shown when its button is clicked?